unitstatushandler/unitstatushandler.go

```diff
@@ -257,6 +263,9 @@ func (instance *Instance) ProcessRunStatus(instances []cloudprotocol.InstanceSta
 
 	instance.initDone = true
 
+	instance.unitStatus.Instances = instances
```

Comment:
This is not needed as you send zero array from launcher.

Reply:
done